content/en/docs/hertz/tutorials/toolkit/toolkit.md

```diff
@@ -0,0 +1,1019 @@
+  string Other = 1[(api.body)="other"];
+}
+
+message OtherResp {
+  string Resp = 1;
+}
+
+service HelloService {
+  rpc Method1(HelloReq) returns(HelloResp) {
+    option (api.get) = "/hello";
+  }
+  rpc Method2(OtherReq) returns(OtherResp) {
+    option (api.post) = "/other";
+  }
+}
+
+service NewService {
+  rpc Method3(OtherReq) returns(OtherResp) {
+    option (api.get) = "/new";
+  }
+}
+```
+
+2. Switch to the directory where the new command was executed and update the modified IDL
+
+```bash
+hz update -I idl -idl idl/hello/hello.proto
+```
+
+3. As you can see
+
+   Add new method under "biz/handler/hello/hello_service.go"<br>
+   The file "new_service.go" and the corresponding "Method3" method have been added under "biz/handler/hello"
+
+Now let's develop the "Method2" interface
+
+```go
+// Method1 .
+// @router /hello [GET]
+func Method1(ctx context.Context, c *app.RequestContext) {
+   var err error
+   var req hello.HelloReq
+   err = c.BindAndValidate(&req)
+   if err != nil {
+      c.String(400, err.Error())
+      return
+   }
+
+   resp := new(hello.HelloResp)
+
+   // You can modify the logic of the entire function, not just the current template
+   resp.RespBody = "hello," + req.Name // added logic
+
+   c.JSON(200, resp)
+}
+
+// Method2 .
+// @router /other [POST]
+func Method2(ctx context.Context, c *app.RequestContext) {
+   var err error
+   var req hello.OtherReq
+   err = c.BindAndValidate(&req)
+   if err != nil {
+      c.String(400, err.Error())
+      return
+   }
+
+   resp := new(hello.OtherResp)
+
+   // added logic
+   resp.Resp = "Other method: " + req.Other
+
+   c.JSON(200, resp)
+}
+```
+
+4. Compile the project
+
+```bash
+go build
+```
+
+5. Run the project and test it
+
+Run the project:
+
+```bash
+./{{your binary}}
+```
+
+Test：
+
+```bash
+curl --location --request POST 'http://127.0.0.1:8888/other' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "Other": "other method"
+}'
+```
+
+If it returns `{"Resp":"Other method: other method"}`, it means that the interface is connected.
+
+### The structure of the generated code
+
+The structure of the code generated by hz is similar. The following is an example of the structure of the code generated by the section "Create a project based on thrift IDL" to illustrate the meaning of the code generated by hz.
+
+```bash
+.
+├── biz                                // business layer, which stores business logic related processes
+│   ├── handler                        // store handler file
+│   │   ├── hello                      // hello/example corresponds to the namespace defined in thrift IDL; for protobuf IDL, it corresponds to the last level of go_package
+│   │   │   └── example
+│   │   │       ├── hello_service.go   // the handler file, the user implements the method defined by the IDL service in this file, and will search for the existing handler in the current file when updating, and append a new handler to the end
```

Comment:
"the user implements" -> "the user will implement"
后面改成"It will search for the existing handler in the current file when "update" command, and append a new handler to the end"

Reply:
fixed.